CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Upgraded build/test images to use Golang 1.13 instead of 1.11 ([#27](https://github.com/cyberark/summon-aws-secrets/issues/27))
```

Comment:
is this a user facing change? is there some value to the user that we should be calling out here?

Reply:
Yeah I added it intentionally because this is possibly a user-visible change because the binaries generated for releases may not behave 1:1 as expected between these versions. I can remove this if needed.